build.gradle

```diff
@@ -41,7 +41,7 @@ msbuild {
 }
 
 nunit {
-    nunitVersion = '3.0.1'
+    nunitVersion = '3.8.0'
```

Comment:
If latest nunit plugin is referenced that would default the nunit console runner to 3.9.0